src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
Something I think of with this project specifically is that with most players, playlists are built into the main player, which makes me want the API to feel more built-in even if this is a plugin. If designating this as a contrib plugin would help that make more sense, so that it's less of an 'unowned object' and more of an optional extension or core, I could get behind that.

One specific concern I have with the current design is the potential confusion between `player.playlist().item()` and `player.playlist.item()`. With everything else being property functions, I think the first is what people will expect.